### PR TITLE
Colors

### PR DESCRIPTION
**Pull request by jmagalhaes-xing**

### CODE REVIEW

File: src/app/core/theming/_theming.scss [hunk 1]
```diff
@@ -1,9 +1,18 @@
-@function keds-default-theme($primary, $secondary, $tertiary, $quaternary, $greyscale) {
+@function keds-palette($base, $main, $hover, $gradient-start, $gradient-end) {
+  @return (
+  main: map-get($base, $main),
+  hover: map-get($base, $hover),
+  gradient: (
+    start: map-get($base, $gradient-start),
+    end: map-get($base, $gradient-end),
+  ),
+  );
```
Comment (RicardoNeves) [inline src/app/core/theming/_theming.scss]: identate?

File: src/app/core/theming/_theming.scss [hunk 2]
```diff
@@ -1,9 +1,18 @@
...
+}
+
+@function keds-default-theme($primary, $accent, $core) {
   @return (
   primary: $primary,
-  secondary: $secondary,
-  tertiary: $tertiary,
-  quaternary: $quaternary,
-  greyscale: $greyscale
+  accent: $accent,
```
Comment (RicardoNeves) [inline src/app/core/theming/_theming.scss]: sort

File: src/app/core/theming/_palette.scss [hunk 3]
```diff
@@ -1,3 +1,20 @@
...
+  500: #195ed1,
+);
+
+$keds-red: (
+  100: #e9004c,
+  200: #e9004c,
+  300: #e9004c,
+  400: #e9004c,
+  500: #e9004c,
+  600: #e9004c,
```
Comment (RicardoNeves) [inline src/app/core/theming/_palette.scss]: should this removed? or maybe we missed this "600" on $keds-blue?

File: src/app/core/theming/_theming.scss (hunk 1 above)
Comment (jmagalhaes-xing) [reply src/app/core/theming/_theming.scss]: The linter makes it put this way, can't really figure out why. Is this a special SASS rule?

File: src/app/core/theming/_palette.scss (hunk 3 above)
Comment (jmagalhaes-xing) [reply src/app/core/theming/_palette.scss]: I was waiting on the other tons of red from Anja, I remove those in the meantime. We're supposed to have at hover and gradient at least.

**Pull request closed without merging**